### PR TITLE
Fix option --opts -> --opt

### PR DESCRIPTION
**Pull request by rhatdan**
Fix typo in podman volume create man page.

#### Does this PR introduce a user-facing change?

```release-note
None
```
Found in https://github.com/containers/podman/discussions/17741

### CODE REVIEW

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Review by flouthoc — APPROVED** [review]
LGTM

**Review by flouthoc — APPROVED** [review]
/lgtm
/hold

**Pull request merged**
